debian/changelog

```diff
@@ -1,3 +1,9 @@
+golang-gocapability-dev (0.0~git20150324-1) UNRELEASED; urgency=low
+
+  * New upstream version (0.0~git20150324).
```

Comment:
For a second I thought this was a mistake, but nope, https://github.com/syndtr/gocapability/commit/8e4cdcb3c22b40d5e330ade0b68cb2e2a3cf6f98 (it really is latest commit today!)